tests/run_tests.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+pushd tests
+
+function finish {
+   popd
+}
+trap finish EXIT
+
+pipenv install
+cargo build --verbose $@
+pipenv run pytest -n auto
```

Comment:
I was checking the travis pipeline and found out that there is this [`tests/.travis-runner.sh`][https://github.com/RustPython/RustPython/blob/master/tests/.travis-runner.sh] that does very similar thing as this script. Can we merge it somehow? e.g. Have only one and skip the install rust part if rust is already present. Or have the `.travis-runner.sh` call this script.

Reply:
Ah also I vaguely remember I had some problem running pytest-xdist in Travis. Could you kindly test it in Travis?